great_expectations/datasource/fluent/serializable_types/pyspark.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import annotations
+
+from great_expectations.compatibility.pyspark import pyspark
+
+
+class SerializableStructType(dict):
+    """Custom type implementing pydantic validation."""
+
+    struct_type: pyspark.sql.types.StructType
+
+    def __init__(
+        self,
+        fields_or_struct_type: pyspark.sql.types.StructType
+        | list[pyspark.sql.types.StructField]
+        | None,
+    ):
+        if isinstance(fields_or_struct_type, pyspark.sql.types.StructType):
+            self.struct_type = fields_or_struct_type
+        else:
+            self.struct_type = pyspark.sql.types.StructType(
+                fields=fields_or_struct_type
+            )
+
+        json_value = self.struct_type.jsonValue()
```

Comment:
@anthonyburdi Just curious, is `jsonValue()` a PySpark method?  Thanks.

Reply:
It is!